.github/workflows/ci.yml

```diff
@@ -47,13 +47,47 @@ jobs:
         run: |
           go test ./... -short
   integration-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        git-version:
+          - 2.20.0 # oldest supported version
```

Comment:
This is going to increase our discipline around flakiness because now there's 6 runs of each test!